app/imports/components/navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default Navbar = ({ isLogged }) => (
+  <nav role="navigation">
+    <ul>
+      <li><a href="/">Home</a></li>
+      { isLogged ? 
```

Comment:
this is just me, although this should be moved out into a sole function that returns the proper elements instead of having it here

there are my other ways to do this